lib/lightning/usage_tracking/github_client.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Lightning.UsageTracking.GithubClient do
+  @moduledoc """
+  A github client to make unauthenticated HTTP requests to Github.
+  """
+  use Tesla, only: [:head], docs: false
+
+  alias Lightning.UsageTracking.ResponseProcessor
+
+  @host "https://github.com/"
+
+  def open_fn_commit?(nil = _commit_sha), do: false
+  def open_fn_commit?("" = _commit_sha), do: false
+
+  def open_fn_commit?(commit_sha) do
+    @host
+    |> build_client
+    |> head("OpenFn/lightning/commit/#{commit_sha}")
+    |> elem(1)
+    |> ResponseProcessor.successful_200?()
```

Comment:
`elem(1)` is used in rare cases. Most of the times we do pattern match to have an idea of the data that is flowing on the pipeline.